scripts/idl/tests/outputs/large_lighting_app/cpp-app/callback-stub.cpp

```diff
@@ -92,6 +67,12 @@ void emberAfClusterInitCallback(EndpointId endpoint, ClusterId clusterId)
     case ZCL_OPERATIONAL_CREDENTIALS_CLUSTER_ID:
         emberAfOperationalCredentialsClusterInitCallback(endpoint);
         break;
+    case ZCL_OTA_SOFTWARE_UPDATE_PROVIDER_CLUSTER_ID:
```

Comment:
It looks like maybe we used to sort by cluster id, not name?  Probably OK....

Reply:
I did not try hard to match ordering since I believe in the end it probably does not matter. As long as we are 'close enough' and functionality seems identical in the end.